### PR TITLE
workaround: Uninstall btrfsmaintenance bsc#1174436

### PR DESCRIPTION
**Pull request by dzedro**
- Related ticket: https://bugzilla.suse.com/show_bug.cgi?id=1174436
- Verification run:
http://10.100.12.155/tests/16952#step/autoyast_reboot/1
https://openqa.suse.de/tests/4960247 qam-allpatterns
https://openqa.suse.de/tests/4960248 qam-gnome

### CODE REVIEW

File: lib/autoyast.pm [hunk 1]
```diff
@@ -59,6 +59,7 @@ sub expand_patterns {
             my @sle15;
             push @sle15, qw(base minimal_base enhanced_base apparmor sw_management yast2_basis);
             push @sle15, qw(x11 gnome_basic fonts)                                                       if check_var('DESKTOP', 'gnome');
+            push @sle15, qw(x11_yast)                                                                    if is_sle('15-sp2+');
```
Comment (coolo) [inline lib/autoyast.pm:62]: I think adding x11_yast if x11 isn't there (aka not DESKTOP=gnome above) is wrong. And IMO you can add that pattern also on ga and sp1. So just merge with the line above

**Comment by coolo** [comment]
And you added and remove WIP at the same time - what's true?

**Comment by dzedro** [comment]
> And you added and remove WIP at the same time - what's true?

WIP is true :) this change does not solve the problem, there are another pattern missing

**Comment by coolo** [comment]
The proper workaround for this bug is removing btrfsmaintenance in autoyast_reboot. Can you try that?

**Comment by dzedro** [comment]
> The proper workaround for this bug is removing btrfsmaintenance in autoyast_reboot. Can you try that?

I will try that.

**Comment by dzedro** [comment]
When btrfsmaintenance is removed gdm does start without permission problems on /dev/tty7

**Comment by coolo** [comment]
I'm all for getting rid of this failure, but possibly this should be specific to SP2? You're tainting now all autoyast installations

**Review by coolo — APPROVED** [review]

**Comment by dzedro** [comment]
> I'm all for getting rid of this failure, but possibly this should be specific to SP2? You're tainting now all autoyast installations

yes, I forgot to set it, changed that to only 15-SP2

**Pull request merged**